packages/jaeger-ui-components/src/types/trace.ts

```diff
@@ -34,6 +34,7 @@ export type TraceLog = {
 };
 
 export type TraceProcess = {
+  processID?: string;
```

Comment:
There does not seem to be any error without this property. It may be just a bug in the testing code that it adds it but does not seem to be needed.

In general it isn't a type issue if you assign to a variable object that has more properties than required.

Reply:
Property removed. https://github.com/grafana/grafana/pull/58464/commits/b60a2e64145cbcf51140d045dc3ec78e9eb7e4ea